File: Lua_5.4/1221 - Primo Rápido.lua
--- author: Josélio de S. C. Júnior <joseliojrx25@gmail.com>
--- copyright: Josélio de S. C. Júnior 2021

local r = io.read()
local function isPrime(n)
  if n == 1 then return 'Not Prime' end
  local i = 2
  while i * i <= n do
    if n % i == 0 then return 'Not Prime' end
    i = i + 1
  end
  return 'Prime'
end
for _ = 1, r, 1 do
  print(isPrime(tonumber(io.read())))
end

File: JavaScript/1221 - Primo Rápido.js
/**
 * @author Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
 * @copyright Josélio Júnior (Lunatic Fox) 2021
*/

const p = require('fs').readFileSync('/dev/stdin', 'utf8');
const l = p.split('\n');

const r = l.shift();

const isPrime = n => {
    if (n === 1) return 'Not Prime';
    
    let i = 2;
    while (i * i <= n) {
        if (n % i === 0) return 'Not Prime';
        i++;
    }
    
    return 'Prime';
};

for (let i = 0; i < r; i++) {
    const x = +l.shift();
    console.log(isPrime(x));
}